src/braket/circuits/instruction.py

```diff
@@ -168,14 +206,29 @@ def copy(
         if target_mapping and target is not None:
             raise TypeError("Only 'target_mapping' or 'target' can be supplied, but not both.")
         elif target is not None:
-            return Instruction(self._operator, target)
+            new_target = target
         else:
-            return Instruction(self._operator, self._target.map(target_mapping or {}))
+            new_target = self._target.map(target_mapping or {})
+        if control_mapping and control is not None:
```

Comment:
_super_ nit: extra space for readability
```suggestion
            new_target = self._target.map(target_mapping or {})

        if control_mapping and control is not None:
```